frontend/src/metabase/visualizations/click-actions/modes/EmbeddingSdkMode.ts

```diff
@@ -0,0 +1,34 @@
+import type { QueryClickActionsMode } from "../../types";
+// import { ColumnFormattingAction } from "../actions/ColumnFormattingAction";
+import { DashboardClickAction } from "../actions/DashboardClickAction";
+import { HideColumnAction } from "../actions/HideColumnAction";
+import { NativeQueryClickFallback } from "../actions/NativeQueryClickFallback";
+
+export const EmbeddingSdkMode: QueryClickActionsMode = {
+  name: "default",
+  hasDrills: [
```

Comment:
should we use a different key for this? I think we should keep `hasDrills` as a boolean and then have a `drills` key that has an array. It'll make the type a little easier to understand

Reply:
saw the change, nicely done 😄